src/Tickets/Commerce/Reports/Data/Order_Summary.php

```diff
@@ -0,0 +1,278 @@
+<?php
+namespace TEC\Tickets\Commerce\Reports\Data;
+
+use TEC\Tickets\Commerce\Status\Completed;
+use TEC\Tickets\Commerce\Status\Status_Handler;
+use TEC\Tickets\Commerce\Ticket;
+use TEC\Tickets\Commerce\Utils\Value;
+use Tribe__Tickets__Ticket_Object as Ticket_Object;
+use Tribe__Tickets__Tickets;
+
+/**
+ * Class Order_Summary.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Reports\Data
+ */
+class Order_Summary {
+	/**
+	 * @since TBD
+	 *
+	 * @var int The post ID.
+	 */
+	protected int $post_id;
+
+	/**
+	 * @since TBD
+	 *
+	 * @var Ticket_Object[] The tickets.
+	 */
+	protected array $tickets = [];
+
+	/**
+	 * @since TBD
+	 *
+	 * @var array<string,array<string,mixed>> The tickets by type.
+	 */
+	protected array $tickets_by_type = [];
+
+	/**
+	 * @since TBD
+	 *
+	 * @var array<string,array{label: string, qty_sold: int, total_sales_amount: float|string, total_sales_price: string}> The event sales by status.
+	 */
+	protected array $event_sales_by_status = [];
+
+	/**
+	 * @since TBD
+	 *
+	 * @var array{qty: int, amount: float|string, price: string} The total sales.
+	 */
+	protected array $total_sales = [];
+
+	/**
+	 * @since TBD
+	 *
+	 * @var array{qty: int, amount: float|string, price: string} The total ordered.
+	 */
+	protected array $total_ordered = [];
+
+	/**
+	 * @since TBD
+	 *
+	 * @var array|array[] The total ordered.
```

Comment:
I do not think this can be an array (`array`) or (`|`) an array of arrays (`array[]`).
Please review and correct.